src/autoUpdate.ts

```diff
@@ -14,9 +14,8 @@
  * limitations under the License.
  */
 import { enums } from '@optimizely/optimizely-sdk';
-import { LoggerFacade } from '@optimizely/js-sdk-logging';
-
 import { ReactSDKClient } from './client';
+import { LoggerFacade } from '@optimizely/optimizely-sdk/dist/modules/logging';
```

Comment:
we should export this from JS sdk entrypoint as well in a future. For now, it's okay.

Reply:
Yup. I agree.  Thanks @raju-opti